From 1eb2226aaf664d0be746753a32f82ee2e04c2f0b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tom=C3=A1=C5=A1=20Hrn=C4=8Diar?= <thrnciar@redhat.com>
Date: Tue, 1 Mar 2022 15:31:54 +0100
Subject: [PATCH] Replace @asyncio.coroutine decorator with async def

In Python 3.11 @asyncio.coroutine decorator was removed and it should
be replaced with async def call.

Fixes: #280
---
 tests/unit/test_cors_config.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/tests/unit/test_cors_config.py b/tests/unit/test_cors_config.py
index 817410e..9fe1052 100644
--- a/tests/unit/test_cors_config.py
+++ b/tests/unit/test_cors_config.py
@@ -29,8 +29,7 @@ async def _handler(request):
 
 class _View(web.View, CorsViewMixin):
 
-    @asyncio.coroutine
-    def get(self):
+    async def get(self):
         return web.Response(text="Done")
 
 
